test/lib/host/gen_helpers.js

```diff
@@ -123,7 +123,7 @@ describe('genHelpers', () => {
       setTimeout(() => {
         assert.ok(eventSeen, 'did not emit test event')
         done()
-      }, 1)
+      }, 20)
```

Comment:
feels like `eventSeen` in both cases is redundant and `done()` callback can be called from within `ev.once('self:test', (...args) => {` no?

Reply:
true, the test itself is a bit off - a event emitter is always async, and the timeout thing is not making a difference